templates/kubernetes-cluster.template

```diff
@@ -977,21 +987,33 @@ Resources:
       - Key: 'kubernetes.io/service-name'
         Value: 'kube-system/apiserver-public'
 
-  # Security group to allow public access to port 443
+  # Security group to allow public access to port 443 and 6443
   ApiLoadBalancerSecGroup:
     Type: AWS::EC2::SecurityGroup
     Properties:
       GroupDescription: Security group for API server load balancer
       VpcId: !Ref VPCID
       SecurityGroupIngress:
-        CidrIp: !Ref ApiLbLocation
-        FromPort: 443
-        ToPort: 443
-        IpProtocol: tcp
+        - CidrIp: !Ref ApiLbLocation
+          FromPort: 443
+          ToPort: 443
+          IpProtocol: tcp
       Tags:
       - Key: Name
         Value: apiserver-lb-security-group
 
+  # Allow the load balancer to receive traffic on port 6443 from the cluster security group
```

Comment:
probably don't need this if the comment inside the metadata is legit

Reply:
👍